src/components/dictionaryConcepts/components/MappingModal.jsx

```diff
@@ -7,15 +7,17 @@ import {
 import InternalMapping from './InternalMapping';
 import ExternalMapping from './ExternalMapping';
 
-class MappingModal extends Component {
+export class MappingModal extends Component {
   constructor(props) {
     super(props);
     this.state = {
       type: 'Internal Mapping',
-      map_type: '',
+      to_concept_url: props.to_concept_url,
+      url: props.url,
+      map_type: props.map_type,
       concept_url: '',
       term: '',
-      source_url: '',
+      source_url: props.to_concept_url,
```

Comment:
You have used `props.*` more than two times, it make sense if you destructure it.

Reply:
Ok, I will do just that, thanks!